caom2pipe/transfer_composable.py

```diff
@@ -95,15 +95,7 @@ class Transfer:
 
     def __init__(self):
         self._client = None
-        self._observable = None
-
-    @property
-    def observable(self):
-        return self._observable
-
-    @observable.setter
-    def observable(self, value):
-        self._observable = value
+        self._logger = logging.getLogger(self.__class__.__name__)
```

Comment:
Can't the logger be defined at the module (`transfer_composable`) level?

Reply:
My personal preference is to have the `class` name in the logging output, but if there's a more Pythonic way to declare loggers, point me to a description.

Reply:
OK